### ISSUE TITLE
future update

### ISSUE THREAD

**Issue by RMPPARTS** [mchristopher/PokemonGo-DesktopMap#1216]
I was able to load this program on a windows 8 tablet and it works perfectly, I can remote in from phone to check map.  I found the information to be 100% accurate for spawn points and times.  But would it be possible to add appearing pokemon to lured pokestops?

**Comment by 3Trace** [comment]
They do currently track that, but only when it's scanned over the location that's lured during the 3 min window the poke is spawned there.

**Comment by DHollande** [comment]
How is the performance on your tablet compared to using it on a desktop or laptop?

**Comment by RMPPARTS** [comment]
I am not sure as I only installed it on a tablet. For reference I just ran a 5 radius scan and it took 10 min to populate all the circles.  I am not sure if the issue/request I listed above may be affected by using a tablet.  At a local park I can sit between 4 lured stops but the only thing that appears on this map is the one spawn point next to this area, it is always correct but the other lured stuff that continuously appear in the game do not show up on the map even if I use a 1 radius that covers all 4 lured stops.
